pages/settings/activity.vue

```diff
@@ -0,0 +1,155 @@
+<template>
+  <v-container>
+    <v-row>
+      <v-col cols="12" :offset-md="1" md="5" class="pt-0 pb-4">
+        <v-list two-line class="mb-2">
+          <v-list-item-group>
+            <v-list-item
+              v-for="file in logFiles"
+              :key="file.Name"
+              :href="getLogFileLink(file.Name)"
+              target="_blank"
+            >
```

Comment:
```suggestion
            <v-list-item
              v-for="file in logFiles"
              :key="file.Name"
              :href="getLogFileLink(file.Name)"
              target="_blank"
              rel="noopener"
            >
```

https://web.dev/external-anchors-use-rel-noopener/